data/places/tei/4130.xml

```diff
@@ -99,13 +99,17 @@
     <revisionDesc status="draft">
       <change who="http://syriaca.org/documentation/editors.xml#wpotter" when="2025-06-18-05:00">CHANGED: Aligned Syriac World records with new places data model and schema</change>
       <change who="http://syriaca.org/documentation/editors.xml#wpotter" when="2020-06-19-05:00">CREATED: place</change>
+      <change who="http://syriaca.org/documentation/editors.xml" when="2025-07-30-12:00">CHANGED: added Syriac placeNames and related bibliographical sources</change>
```

Comment:
This is great! You can add your specific editor ID you shared with us: mgbilby, so the `@who` would be http://syriaca.org/documentation/editors.xml#mgbilby